dso-l2/src/main/java/com/tc/net/groups/Node.java

```diff
@@ -19,23 +19,33 @@
 package com.tc.net.groups;
 
 
+import java.util.Objects;
+
+import static java.util.Objects.requireNonNull;
+
 public class Node {
 
+  private static final String DELIMITER = ":";
+
   private final String host;
   private final int    port;
   private final int    groupPort;
   private final int    hashCode;
 
+  public Node(String hostPort) {
+    this(getHost(hostPort), getPort(hostPort));
+  }
+
```

Comment:
are you just adding this for code reuse convenience in passing host:port?

Reply:
yes